main.go

```diff
@@ -4,12 +4,14 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"strconv"
 
 	"github.com/birdsean/review-droid/comments"
 	"github.com/birdsean/review-droid/github_client"
 	"github.com/birdsean/review-droid/openai"
 	"github.com/birdsean/review-droid/transformer"
 	"github.com/google/go-github/v53/github"
+	openai_api "github.com/sashabaranov/go-openai"
```

Comment:
Refactor Suggestion: There are two openai packages being imported. Remove "openai" and only use "openai_api".